charts/localstack/values.yaml

```diff
@@ -130,9 +130,18 @@ service:
 ingress:
   enabled: false
   annotations: {}
-    # Adjust the ingress class when not using nginx as incress controller
+    # Adjust the ingress class when not using nginx as ingress controller
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
+
+    # If you are using SSL passthrough for the certificates for nginx, uncomment the
+    # annotations below (note: nginx needs to be started with `--enable-ssl-passthrough` for these to work)
+    # nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    # nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    # nginx.ingress.kubernetes.io/ssl-redirect: "true"
```

Comment:
Since you migrate the manifest to use the new `ingressClass` we should remove this section.
And imho the nginx specific comments don't really belong here as well. I would suggest to remove the whole section of comments.
```suggestion
```

Reply:
Now that the preceding comment block is removed I'm thinking the original comment wording made less sense so this has been updated and the field is now uncommented but with an empty value rather than defined as `nginx`